packages/docusaurus-theme-classic/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "lib": ["DOM"],
+    "lib": ["DOM", "ESNext"],
```

Comment:
replacing this with ES2019, as it will run in NodeJS 12 (unlike any new ES2020 feature)

However not yet 100% convinced it's the right move because some ES2019/older features don't have all a very good browser support, so we'll have to remember to not use them